classes/fields/pick.php

```diff
@@ -726,6 +726,34 @@ public function schema( $options = null ) {
 
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$value" missing

<sub>hash: c67528e14c42bbc255c1ffb24b574be1</sub>